app/templates/index.html

```diff
@@ -29,7 +29,9 @@
   <ul>
     {% for pref in current_user.preferences %}
     <li class="preference" data-id="{{ pref.id }}">
+      {% if pref.food.lower() in upcoming %}<strong>{% endif %}
```

Comment:
Probably better to do this as a css class so you can get away with just one check of the condition, but it doesn't really